src/Umbraco.Web/UmbracoApplicationBase.cs

```diff
@@ -15,7 +14,14 @@ namespace Umbraco.Web
     /// </summary>
     public abstract class UmbracoApplicationBase : HttpApplication
     {
-        private IRuntime _runtime;
+        // needs to be static, if and exception is thrown in Application_Start the current HttpApplication is discarded
+        // and a new instance is created, meaning we loose value if it's already been set.
+        private static IRuntime _runtime;
```

Comment:
This static ref is the main thing that's worrisome. Is there any way you can think of that would have this cause problems by pinning this statically? Off the top of my head I can't think of any since IRuntime is a singleton anyways, if the boot fails, it fails until the app is restarted right? Any other concerns about this you can think of?

Reply:
I've reverted the change since I didn't end up needing the `IRuntime` anyways